content_creator.py

```diff
@@ -0,0 +1,33 @@
+from image_loader import ImageLoader
+from quote_loader import QuoteLoader
+from message_sender import MessageSender
+import logging
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+logger = logging.getLogger("ContentCreationLogger")
+
+
+class ContentCreator:
+    def __init__(self, quote_api_url, image_api_url, webhook_url, author_name):
+        self.quote_loader = QuoteLoader(quote_api_url)
+        self.image_loader = ImageLoader(image_api_url)
+        self.message_sender = MessageSender(webhook_url)
+        self.author_name = author_name
+
+    def create_content(self):
+        quote = self.quote_loader.get_quote()
+        image = self.image_loader.get_image()
+        if quote and image:
```

Comment:
Here you can use 'guard clause', it makes code a bit more cleaner and readable.

```python
if not (quote and image):
    logger.error("Failed to create content")
    return None

logger.info("Content is created")
return f"Sent by {self.author_name}\n\nQuote: {quote}\n\n![Image]({image})"
```

Lets imagine, if image loading failed, you will receive a lot of errors ('failed to load image' first, then 'failed to create content', and finally 'nothing to sent to microsoft teams'), but in fact it was the only one error -  with loading image. So I would also suggest to get rid of this check and introduce exceptions at lower steps instead of returning `None` (getting quote and image url), because no matter what, if any single step fails, entire script should fail too, so you aclually can be not care of all consequents steps and raise exception straight without catching it.

Reply:
fixed